std/datetime.d

```diff
@@ -31104,6 +31123,7 @@ For terms of use, see http://www.unicode.org/copyright.html
     Params:
         tzName = The TZ Database name to convert.
   +/
+deprecated("Use parseTZConversions instead")
```

Comment:
We don't normally put the date for removal in the deprecation message, but I did forget to add it in the ddoc comments, which I fixed.